scalafmt-tests/src/test/scala/org/scalafmt/FormatTests.scala

```diff
@@ -74,10 +81,7 @@ class FormatTests extends FunSuite with CanRunTests with FormatAssertions {
       )
       .get
     debug2.printTest()
-    assertNoDiff(formattedAgain, obtained, "Idempotency violated")
-    if (!onlyManual) {
-      assertNoDiff(obtained, t.expected)
-    }
+    assertEquals(formattedAgain, obtained, "Idempotency violated")
```

Comment:
looks like we're also changing one type of assert to another. can we do that in a separate pr? i think it deserves its own commit, with explicit explanation.

Reply:
Moved the change to https://github.com/scalameta/scalafmt/pull/2312